rudder-web/src/main/scala/com/normation/rudder/web/snippet/administration/PropertiesManagement.scala

```diff
@@ -150,7 +155,11 @@ class PropertiesManagement extends DispatchSnippet with Loggable {
       Run(
         s"""
             $$("#mandatory").prop("disabled",${!newStatus});
-            $$("#explanation").prop("disabled",${!newStatus});"""
+            $$("#explanation").prop("disabled",${!newStatus});
+            $$(document).ready(function(){
```

Comment:
You can use lift thing to do "document.ready" instread of pure javascript (OnLoad If I remeter correctly)
And add a comment to explain why it's done here and it will apply to the whole page and not only "change message"